### ISSUE TITLE
[Accessibility]A11y_VSCode_UserSnippets_keyboard: User Snippets treeview closes on tab keypress.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52042]
**Environment Details:** 
VSCode Version : 1.24.0
OS Version : Win10

**Additional Details:**
MAS Violated: MAS2.4.3

Repro Steps:
1. Launch VS Code while NVDA is ON.
2.Navigate to Settings icon in Activity Bar->Press Enter->Select "User Snippets" from dropdown..

Actual:
When user clicks on "Tab" key then opened "User Snippets" treeview gets close.

Expected:
When user clicks on "Tab" key then focus should go on expanded dropdown values instead of closing the treeview.

User Impact:
User will get confused on which key the proper action will perform.

MAS Reference:
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9}

Attachment for reference: 
[A11y VSCode Problems NVDA treeviewcloses.pptx](https://github.com/Microsoft/vscode/files/2106117/A11y.VSCode.Problems.NVDA.treeviewcloses.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by RMacfarlane** [comment]
The same control is opened in many different scenarios (whenever user input is requested) and has the same tab behavior. I'm going to close all but one instance of this problem since the root cause is one control. 
(https://github.com/Microsoft/vscode/issues/51854)
